routes/discordactions.js

```diff
@@ -16,6 +18,7 @@ const router = express.Router();
 router.post("/groups", authenticate, checkIsVerifiedDiscord, validateGroupRoleBody, createGroupRole);
 router.get("/groups", authenticate, checkIsVerifiedDiscord, getAllGroupRoles);
 router.post("/roles", authenticate, checkIsVerifiedDiscord, validateMemberRoleBody, addGroupRoleToMember);
+router.post("/nickname", authenticate, checkIsVerifiedDiscord, changeNicknameOfUsers);
```

Comment:
should it be post or patch as you are updating a resource partially

Reply:
with  patch it shows not found and I referred from another api they are post and called put request inside it. so followed the same.